chemprop/cli/hpopt.py

```diff
@@ -43,16 +43,16 @@
     DEFAULT_SEARCH_SPACE = {
         "activation": tune.choice(categories=list(Activation.keys())),
         "aggregation": tune.choice(categories=list(AggregationRegistry.keys())),
-        "aggregation_norm": tune.quniform(lower=1, upper=200, q=1),
-        "batch_size": tune.loguniform(lower=16, upper=256, base=2),
-        "depth": tune.quniform(lower=2, upper=6, q=1),
+        "aggregation_norm": tune.qrandint(lower=1, upper=200, q=1),
+        "batch_size": tune.choice(np.logspace(start=4, stop=8, num=5, base=2)),
```

Comment:
Can this be rewritten using `tune`'s [`qlograndint`](https://docs.ray.io/en/latest/tune/api/doc/ray.tune.qlograndint.html#ray-tune-qlograndint) method?

Reply:
I tried that but it did not exclusively sample powers of 2. It is probably just doing a log distribution over all integers.

Reply:
Here is an example run:
`>>> from ray import tune
>>> sampler = tune.qlograndint(lower=2,upper=8,base=2,q=1)
>>> sampler.sample()
2
>>> sampler.sample()
4
>>> sampler.sample()
7`

Reply:
I like how this turned out later on